testing-common/src/main/groovy/io/opentelemetry/auto/test/base/HttpClientTest.groovy

```diff
@@ -42,7 +42,7 @@ import static org.junit.Assume.assumeTrue
 abstract class HttpClientTest extends AgentTestRunner {
   protected static final BODY_METHODS = ["POST", "PUT"]
   protected static final CONNECT_TIMEOUT_MS = 1000
-  protected static final BASIC_AUTH_KEY = "custom authorization header"
+  protected static final BASIC_AUTH_KEY = "custom-authorization-header"
```

Comment:
In this PR PlayJavaWEClientTest fails because of invalid header name characters (space) which looks correct to me... The Play tests have a couple other assertions that fail (TimeoutException vs ConnectException) that seem similarly like they should be failing, as if on current master we're not actually running these tests so I need to investigate more what's going on...

Reply:
> as if on current master we're not actually running these tests

oh, wow, they were not, before this PR!!!

cc: @tylerbenson

Reply:
@anuraaga could you elaborate... which task isn't failing that should be?  Any idea why they aren't being executed?

Reply:
I didn't end up digging on the why, but I think it was this test that started failing with this PR

https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/874b157fe5fdaad80fd82f3ece0600b6860924e7/instrumentation/play-ws/play-ws-2.1/src/test/groovy/PlayWSClientTest.groovy

Netty threw a validation error because of the spaces in this header name which seems right to me.